src/ui/tray-icon.cpp

```diff
@@ -444,30 +441,6 @@ void SeafileTrayIcon::showSettingsWindow()
     seafApplet->settingsDialog()->activateWindow();
 }
 
-void SeafileTrayIcon::onActivated(QSystemTrayIcon::ActivationReason reason)
-{
-#if !defined(Q_OS_MAC)
-    switch(reason) {
-    case QSystemTrayIcon::Trigger: // single click
-    case QSystemTrayIcon::MiddleClick:
-    case QSystemTrayIcon::DoubleClick:
-        onClick();
-        break;
-    default:
-        return;
-    }
-#endif
-}
-
-void SeafileTrayIcon::onClick()
-{
-    if (state_ == STATE_HAVE_UNREAD_MESSAGE) {
-        viewUnreadNotifications();
-    } else {
-        showMainWindow();
-    }
-}
-
 void SeafileTrayIcon::disableAutoSync()
```

Comment:
Removing all these code would cause seafile-applet does nothing when the user left clicks the tray icon (on windows).